assets/scss/pages/_frontpage.scss

```diff
@@ -0,0 +1,10 @@
+.foo {
+  transition: 150ms ease-in-out;
+  color: black;
+  transform: translateX(15px);
+
+  &.bar {
+    transform: translateX(30px);
+    color: red;
+  }
+}
```

Comment:
since we're using the extension on the modal transition now, I don't think we need these here as an example
```suggestion
```